.gitlab-ci.yml

```diff
@@ -28,6 +28,7 @@ default:
 
 before_script:
   - yarn config set ignore-engines true
+  - yarn config set workspaces-experimental true
```

Comment:
What is this voodoo?

Reply:
this actually doesn't fix the issue. workspaces is not enabled by default. the real issue was a circular dep